### PR TITLE
Adding SALT to the observatories.json file

### PR DESCRIPTION
**Pull request by EricDepagne**

### CODE REVIEW

File: astroplan/data/observatories.json [hunk 1]
```diff
@@ -381,4 +381,18 @@
         "name": "mro",
         "source": "MRO website"
     }
-}
\ No newline at end of file
+        "SALT" : {
```
Comment (bmorris3) [inline astroplan/data/observatories.json]: This is picky, but could you correct the indentation on this line and include a source for this information? It's ok to say "SALT website" as your source.

File: astroplan/data/observatories.json [hunk 2]
```diff
@@ -381,4 +381,18 @@
         "name": "mro",
         "source": "MRO website"
     }
```
Comment (bmorris3) [inline astroplan/data/observatories.json]: Need a comma here.

File: astroplan/data/observatories.json [hunk 3]
```diff
@@ -381,4 +381,18 @@
...
\ No newline at end of file
+    "SALT": {
+        "source" : "SALT website",
+        "elevation": 1700, 
+        "name": "South African Large Telescope", 
+        "longitude_unit": "degree", 
+        "latitude_unit": "degree", 
+        "latitude": 20.810499, 
+        "elevation_unit": "meter", 
+        "longitude": 32.561633,
```
Comment (bmorris3) [inline astroplan/data/observatories.json]: [Wikipedia](https://en.wikipedia.org/wiki/Southern_African_Large_Telescope) says that the position is 32°22′33.62″S 20°48′38.44″ E – did you swap the latitude and longitude? Also, eastern longitudes are negative.

**Comment by EricDepagne** [comment]
Maybe I'll manage to do that properly, in the end?

File: astroplan/data/observatories.json (hunk 3 above)
Comment (bsipocz) [inline astroplan/data/observatories.json]: Are they? Shouldn't they be the other way around, following most geography standards?

**Comment by eteq** [comment]
@EricDepagne @bmorris3 - one thing to clarify about this: once astropy v1.1 has been release we will probably be transitioning astroplan over to using the astropy/astropy#4042 machinery.  So assuming that happens, the main data source will be the astropy data server, which you can update at

https://github.com/astropy/astropy-data

So you should probably submit this there (or maybe both there *and* here for good measure).

**Comment by bmorris3** [comment]
Closed by #168.

**Pull request closed without merging**